src/components/Banners/HeroBanner.module.css

```diff
@@ -0,0 +1,30 @@
+.HeroBanner {
+  display: flex;
+
+}
+
+.HeroBanner_img {
+  position: relative;
+  height: 100vh;
```

Comment:
Jesteś pewna, że tutaj powinno być 100vh? To powoduje, że na różnych urządzeniach postacie będą zniekształcone